### PR TITLE
SectorTermDao added.

### PR DESCRIPTION
**Pull request by miltomasz**
An example of PR for Code Review.

### CODE REVIEW

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java [hunk 1]
```diff
@@ -5,34 +5,83 @@
...
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
 @Entity(tableName = "sector_terms", indices = {@Index(value = {"sectorType"})})
 public class SectorTerm {
+    private static final  int greenSectorType = 1;
+    private static final  int blueSectorType = 2;
+    private static final  int yellowSectorType = 3;
```
Comment (miltomasz) [inline app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java]: Change the above constants into enum variables. You may also override toString() method which will return strings like "GREEN", "BLUE" etc. We're on Android but in this case enums should not cause any overhead.

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java [hunk 2]
```diff
@@ -5,34 +5,83 @@
...
-        this.term = term;
+    public SectorTerm(String term, int termType) {
+        String dateFormat = "yyyy.MM.dd";
+        Date dateTerm = new Date();
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(dateFormat);
+        try {
+            dateTerm = simpleDateFormat.parse(term);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
```
Comment (miltomasz) [inline app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java:44]: Try not to use in constructor any logic. In terms of date formatting Room provides TypeConverters. Please check Android Architecture Components codelabs for more details.

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java (hunk 1 above)
Comment (gbielanski) [reply app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java]: Changed to Enum. Good point about toString but I checked default implementation and works fine no need to override.

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java (hunk 2 above)
Comment (gbielanski) [reply app/src/main/java/pl/example/android/garbageapp/data/database/SectorTerm.java:44]: This constructor is not used by Room but only for fake data creation. This is why it is marked as "@ Ignore" but I know what you mean I will add DateConverter in next commit with SectorTermsDatabase class to convert long (timestamp) to Date.

**Comment by miltomasz** [comment]
Changes after code review accepted.

**Pull request merged**